notebook/static/notebook/js/actions.js

```diff
@@ -444,7 +451,7 @@ define(function(require){
                 return true;
             }
         },
-        'move-cursor-up-or-previous-cell':{
```

Comment:
Why do you remove `or-previous-cell` ? This will definitively not be use through command palette, and only in API (even if it will be **shown** in CP), so why not be explicit about what it does ? Or we will get confused user where it will  get unsuspecting behavior like changing the cell selection.